core/input/input_map.cpp

```diff
@@ -516,6 +516,7 @@ const HashMap<String, List<Ref<InputEvent>>> &InputMap::get_builtins() {
 	// ///// UI Text Input Shortcuts /////
 	inputs = List<Ref<InputEvent>>();
 	inputs.push_back(InputEventKey::create_reference(Key::SPACE | KeyModifierMask::CTRL));
+	inputs.push_back(InputEventKey::create_reference(Key::I | KeyModifierMask::CMD_OR_CTRL));
```

Comment:
List the modifier first (it acts the same, but is more consistent with the rest of the codebase):

```suggestion
	inputs.push_back(InputEventKey::create_reference(KeyModifierMask::CMD_OR_CTRL | Key::I));
```